.github/workflows/ci.yaml

```diff
@@ -154,7 +154,7 @@ jobs:
         with:
           fetch-depth: 0 # Shallow clones should be disabled for a better relevancy of analysis
       - name: Set up JDK 17
-        uses: actions/setup-java@c5195efecf7bdfc987ee8bae7a71cb8b11521c00 # v4
+        uses: actions/setup-java@dded0888837ed1f317902acf8a20df0ad188d165 # v4
```

Comment:
mh documentation said dependabot would update the version comments after the hashes :/

https://github.com/dependabot/dependabot-core/issues/4691